dmscripts/notify_suppliers_of_brief_withdrawal.py

```diff
@@ -1,11 +1,16 @@
-from dmutils.email.helpers import hash_string
+from itertools import chain
+
+from dmutils.email.helpers import get_email_addresses, hash_string
 
 from dmutils.env_helpers import get_web_url_from_stage
 
 
 def get_brief_response_emails(data_api_client, brief_id):
     responses = data_api_client.find_brief_responses_iter(brief_id=brief_id, status="submitted")
-    return [response["respondToEmailAddress"] for response in responses]
+    return list(chain.from_iterable(
```

Comment:
what's the advantage of using chain here?

Reply:
What's the alternative? Other than nested `for`s which I find unclear.

Reply:
Does `[]` not work?

Reply:
Oh wait it needs to be flattened I get it

Reply:
Not if the contents are an iterable - to be clear, here I'm having to flatten an iterable of iterables, that's what the `chain.from_iterable` is about.

Reply:
I guess you could put the generator expression inside the call arguments